src/main/java/com/sericulture/authentication/service/UserInfoDetails.java

```diff
@@ -25,6 +28,9 @@ public UserInfoDetails(UserInfo userInfo) {
         //password = userInfo.getPassword();
         marketId = userInfo.getMarketId();
         userMasterId = userInfo.getUserMasterId();
+        if(Objects.nonNull(userInfo.getRole()) && Objects.nonNull(userInfo.getRole().getRoleName())) {
```

Comment:
⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.sizes.LineLengthCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line is longer than 100 characters (found 102).